### PR TITLE
fix: username customization property

### PR DESCRIPTION
**Pull request by pradeepdewda-okta**
## Description:
1. removed extra explain introduced in IDPDiscoveryForm

## PR Checklist

- [ ] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Screenshot/Video:
Screen Shot 2020-09-16 at 11 52 39 AM
Screen Shot 2020-09-16 at 11 53 03 AM
Screen Shot 2020-09-16 at 11 53 12 AM

### Issue:

- [OKTA-306126](https://oktainc.atlassian.net/browse/OKTA-306126)

### CODE REVIEW

File: src/views/idp-discovery/IDPDiscoveryForm.js [hunk 1]
```diff
@@ -31,7 +30,6 @@ export default PrimaryAuthForm.extend({
       className: 'margin-btm-30',
       label: loc('primaryauth.username.placeholder', 'login'),
       'label-top': true,
-      explain: Util.createInputExplain('primaryauth.username.tooltip', 'primaryauth.username.placeholder', 'login'),
       'explain-top': true,
```
Comment (haishengwu-okta) [inline src/views/idp-discovery/IDPDiscoveryForm.js:33]: looks like can also remove
- `explain-top`
- `label-top`
- `label`

Comment (pradeepdewda-okta) [reply src/views/idp-discovery/IDPDiscoveryForm.js]: done.

**Review by gowthamidommety-okta — APPROVED** [review]
lgtm!

**Comment by haishengwu-okta** [comment]
closed in https://github.com/okta/okta-signin-widget/pull/1432

**Pull request closed without merging**